components/post_view/commented_on/commented_on.jsx

```diff
@@ -59,14 +54,18 @@ export default class CommentedOn extends PureComponent {
     render() {
         const username = this.makeUsername();
         const message = this.makeCommentedOnMessage();
+        const parentPostUser = this.props.parentPostUser;
+        const parentPostUserId = parentPostUser && parentPostUser.id;
 
-        const name = (
-            <a
-                className='theme'
-                onClick={this.handleOnClick}
-            >
-                {username}
-            </a>
+        const parentUserProfile = (
+            <UserProfile
+                user={parentPostUser}
+                userId={parentPostUserId}
+                displayName={username}
+                hasMention={true}
+                disablePopover={false}
+                showGuestBadge={false}
```

Comment:
what happens if the parent post is created by a guest account? shouldn't we show it?

Reply:
This was leftover from cleaning done earlier. I'll update the snaps also

Reply:
This `showGuestBadge` flag was leftover from past cleaning. I'll update in the snaps too.